src/utils/timer.cpp

```diff
@@ -94,4 +93,9 @@ void Timer::stop()
         t.join();
 }
 
-} // namespace multipass::utils
+mpu::TimerState mpu::Timer::current_state()
```

Comment:
```suggestion
mpu::TimerState mpu::Timer::state()
```

Reply:
I committed in my branch to avoid the cla check failure.